README.md

```diff
@@ -51,15 +51,18 @@ For detailed guide on how to test and configure, please refer to [INTEGRATIONS.m
 
 ## Pre-requisites
 
-- UNIX-based OS (Linux/Mac) - for
-  Windows, [try this](https://github.com/martomi/chiadog/issues/7#issuecomment-815544302).
+- Linux, MacOS, Windows
 - Python 3.7+
 - Enabled `INFO` logs on your chia farmer
 
 ### How to enable INFO logs on chia farmer?
 
 First configure the log level to `INFO`. This ensures that all logs necessary for chiadog to operate are available
-under `~/.chia/mainnet/log/debug.log`.
+under `~/.chia/mainnet/log/debug.log`. (Windows: `C:\Users\<username>\.chia\mainnet\log\debug.log`)
+
+Windows: The below commands are equivalent on Windows. But you need go to this
+folder `cd C:\Users\leon\AppData\Local\chia-blockchain\app-1.1.0\resources\app.asar.unpacked\daemon` and execute
```

Comment:
Better use `<username>` instead of `leon` here as well

Reply:
Good catch, thanks! 😅